src/org/zaproxy/zap/extension/httpsinfo/SSLServer.java

```diff
@@ -632,7 +638,9 @@ static String doSHA1(byte[] buf, int off, int len) {
 			for (byte b : hv) {
 				f.format("%02x", b & 0xFF);
 			}
-			return f.toString();
+			String retVal=f.toString();
+			f.close();
```

Comment:
Sorry to be obtuse but what would that look like here? Since the use of the formatter is inside a loop which is inside of the try, I'm not sure how to set it up...